app/src/main/java/com/courseproject/tindar/ui/matchlist/MatchlistFragment.java

```diff
@@ -0,0 +1,37 @@
+package com.courseproject.tindar.ui.matchlist;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.courseproject.tindar.databinding.FragmentMatchlistBinding;
+
+public class MatchlistFragment extends Fragment {
```

Comment:
Can we change Matchlist to MatchList? I think Match List is two words.

Reply:
updated